docs/customization.md

```diff
@@ -174,8 +174,13 @@ This content appears at the bottom of every page's main content. More info for t
 
 Any HTML added to this file will be inserted before the closing `<head>` tag. This might include additional `<meta>`, `<link>`, or `<script>` tags.
 
-#### Example
-{: .no_toc }
+Note that by default, this file has the following contents:
+
+```html
+<link rel="shortcut icon" href="{{ 'favicon.ico' | relative_url }}" type="image/x-icon">
```

Comment:
Two differences between the above default `<link>` and the example of a custom favicon below:
1. The order of the attributes `href` and `type`.
2. The use of `relative_url` vs. `{{site.baseurl}}`.
The differences aren't significant, but they might puzzle/worry novice users, and I think they look a bit untidy.

Reply:
Great suggestion - I'll keep them consistent.